package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bufferapp/components",
-  "version": "0.1.7",
+  "version": "0.1.8",
```

Comment:
I think it's best to keep version bumps outside of commits here. This can cause conflicts when merging multiple branches.